app.js

```diff
@@ -3,18 +3,12 @@
 const express = require('express');
 const app = express();
 const morgan = require('morgan');
-const path = require('path');
-module.exports = app; 
+module.exports = app;
 
 app.use(morgan('dev'));
-app.use(express.static(path.join(__dirname, './public')));
```

Comment:
Unneeded for this assignment (however, correct if you intend to statically serve up files in a `public` directory).